### DISCUSSION TITLE
Versioning scheme, why is 0.18.2 not 0.19.0?

### DISCUSSION THREAD

**Discussion by m0wlheld** [Koenkk/zigbee2mqtt#6910] [General]
Although I didn't fall for the April Fool's joke: Why is version 0.18.2 not version 0.19.0 - or asked differently: According to which criteria are major and minor version changed?

**Comment by m0wlheld** [comment]
.. or did i fall for the April Fool's joke?

**Reply by nurikk** [reply]
https://github.com/Koenkk/zigbee2mqtt/discussions/6908#discussioncomment-556244

**Comment by Koenkk** [comment]
Z2M follows semver: https://semver.org/

Which comes down to:
- Major version increase: when there is a breaking change
- Minor version change: when new features get added
- Patch version change: improvements/bug fixes
